pyproject.toml

```diff
@@ -127,20 +127,19 @@ dependencies = [
     "beautifulsoup4",
     "boto3",
     "faker",
-    "GDAL",
```

Comment:
gdal is required for dcpy, but it's just too messy with the annoying versioning issues around alignment of binary version and python version to have it listed here